base/irrationals.jl

```diff
@@ -104,6 +104,7 @@ end
 isfinite(::Irrational) = true
 isinteger(::Irrational) = false
 iszero(::Irrational) = false
+isinteger(::Irrational) = false
```

Comment:
Right, it's been added in another PR since I opened this one. I've just removed it.